services/web-app/config/redirects.mjs

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Redirects previously used by the carbon-website repository.
+ * @see https://github.com/carbon-design-system/carbon-website/blob/main/conf.d/rewrite.conf
+ * @see https://github.com/carbon-design-system/carbon-website/blob/main/gatsby-node.js
+ */
+const websiteRedirects = [
+  // Guidelines rewrites
+  ['/guidelines/iconography/:slug*', '/guidelines/icons/:slug*'],
+  ['/guidelines/layout', '/guidelines/2x-grid/basics'],
+  ['/guidelines/content/glossary', '/guidelines/content/action-labels'],
+  // Experimental -> Cloud, Data, and AI
+  [
+    '/experimental/cards/:slug*',
+    'https://pages.github.ibm.com/CDAI-design/CDAI-design-website/components/card/:slug*'
+  ],
+  [
+    '/experimental/dashboard-widgets/:slug*',
+    'https://pages.github.ibm.com/CDAI-design/CDAI-design-website/components/widget/:slug*'
+  ],
+  [
+    '/experimental/order-summary-template/:slug*',
+    'https://pages.github.ibm.com/CDAI-design/CDAI-design-website/patterns/order-summary/:slug*'
+  ],
+  // Experimental -> community
+  ['/experimental/:slug*', '/community/patterns/:slug*'],
+  // Community index
+  ['/community/components', '/community/component-index'],
+  // IA updates 08/2020
+  ['/get-started/about-carbon', '/all-about-carbon/what-is-carbon'],
+  ['/contributing/governance', '/all-about-carbon/how-we-work'],
+  ['/get-started/design/:slug*', '/designing/get-started/kits/:slug*'],
```

Comment:
```suggestion
  ['/get-started/design/:slug*', '/designing/get-started'],
```
I think just redirecting to the general get started here will work? Current redirect breaks, we don't have a /kits

Reply:
I updated so `http://localhost:3000/get-started/design/sketch` redirects to `http://localhost:3000/designing/sketch` with:

`['/get-started/design/:slug*', '/designing/:slug*']`.